JellyfinPlayer/ItemView/ItemView.swift

```diff
@@ -55,6 +55,39 @@ private struct ItemView: View {
         }
     }
 
+    @ViewBuilder
+    var toolbarItemContent: some View {
+        switch viewModel.item.itemType {
+        case .season:
+            Menu {
+                Button {
+                    (viewModel as? SeasonItemViewModel)?.routeToSeriesItem()
```

Comment:
I understand why we need this pattern but I'm not that _comfortable_ with it. If this is the route we have to go that's okay but I want to see if I can do anything different. Would you mind making your branch public so that I can play with it for just a few minutes?

Reply:
It would be nice if there was a better way 😆 
It is already public to 'PangMo5/ItemView-Navigation' in 'jellyfin/Swiftfin' and is feel free to fix.

Reply:
Yeah, I'm too lazy to try anything. LGTM 👍 

In the future, I would recommend having work on your own branch and it's okay if it was an accident. This one was on the project's `upstream`.